myapp/routes/registration-route.js

```diff
@@ -0,0 +1,106 @@
+var express = require('express');
+var nodemailer = require('nodemailer');
+var router = express.Router();
+var db  = require('../database/config.js');
+
+
+// to display registration form 
+router.get('/register', function(req, res, next) {
+  res.render('registration-form');
+});
+
+// to store user input detail on post request
+router.post('/register', function(req, res, next) {
+    
+    inputData ={
+        name: req.body.name,
+        email: req.body.email,
+        password: req.body.password,
+        user_type:"general",
+    }
+    
+    // check unique email address
+var sql='SELECT * FROM Users WHERE email =?';
```

Comment:
What is difference between **index.js's** `register route` and this `register route`?